src/ruby/ext/symengine/extconf.rb

```diff
@@ -0,0 +1,74 @@
+require 'mkmf'
```

Comment:
mkmf stands for [MakeMakeFile](http://ruby-doc.org/stdlib-2.0.0/libdoc/mkmf/rdoc/MakeMakefile.html). It is used to generate the makefile for the extensions. Even a bare minimum `extconf.rb`([like this](https://github.com/change/filesets/blob/387c6992aee7c12afad54ae1e0d7cb32e4386d19/ext/filesets/extconf.rb)) needs this.